src/test/java/athena/task/taskfilter/ForecastFilterTest.java

```diff
@@ -24,37 +30,39 @@ void testIsTaskIncluded_all_returnsTrue() {
     }
 
     /**
-     * Checks if task is included after applying the week forecast filter.
+     * Checks if task in this week is included after applying the week forecast filter.
      */
     @Test
-    void testIsTaskIncluded_week_returnsTrue() {
+    void testIsTaskIncluded_filterByWeek_returnsTrue() {
+        String todayDateString = LocalDate.now().toString();
         ForecastFilter forecastFilter = new ForecastFilter(Forecast.WEEK);
         Task inputTask = new Task("testName", "0900", "1", "05-11-2020",
-                "16-10-2020", Importance.LOW, "testNotes", 0); // Tested on 12-10-2020
+                todayDateString, Importance.LOW, "testNotes", 0);
```

Comment:
This test is exactly the same as `testIsTaskIncluded_filterByToday_returnsTrue()`. Perhaps a better input could be a day that lies in the same week as `LocalDate.now` and adding this as a parameter.